iOS-School-02_stack.playground/Contents.swift

```diff
@@ -1,33 +1,78 @@
+//MARK: - Stack
```

Comment:
```suggestion
//  MARK: - Stack
```

Check the rest please

Reply:
Fixed.

Reply:
Sorry, I did a typo there.
I meant to have only one space after `//` characters.

Please fix it.

Reply:
Fixed.